### PR TITLE
Update to 4.94.0

### PR DESCRIPTION
**Pull request by pvuorela**
First commit removes old source, ending up lots of changed lines, but the beef in the second commit. See that for details.

And yea, could use a mirror too :) 

@Tomin1 @mlehtima

### CODE REVIEW

File: rpm/lsof.spec [hunk 1]
```diff
@@ -1,53 +1,31 @@
...
-Summary:   Documentation for %{name}
-Group:     Documentation
-Requires:  %{name} = %{version}-%{release}
-Obsoletes: %{name}-docs
-
-%description doc
-Man page for %{name}.
-
 %prep
 %setup -q -n %{name}-%{version}/%{name}
```
Comment (okodron) [inline rpm/lsof.spec]:
```
%autosetup -n %{name}-%{version}/%{name}
```

File: rpm/lsof.spec [hunk 2]
```diff
@@ -1,53 +1,31 @@
...
 
 %build
-LSOF_VSTR=2.6.16 LINUX_BASE=/proc ./Configure -n linux
+./Configure -n linux
 
 make DEBUG="$RPM_OPT_FLAGS" %{?_smp_mflags}
 
 %install
 rm -rf ${RPM_BUILD_ROOT}
```
Comment (okodron) [inline rpm/lsof.spec]: Not necessary any more.

File: rpm/lsof.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/lsof.spec]: Included the common -p1 too, but yea.

File: rpm/lsof.spec (hunk 2 above)
Comment (pvuorela) [reply rpm/lsof.spec]: Removed.

**Review by mlehtima — APPROVED** [review]

File: .gitmodules [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[submodule "lsof"]
+	path = lsof
+	url = https://github.com/lsof-org/lsof.git
```
Comment (xfade) [inline .gitmodules:3]: Mirror: https://github.com/sailfishos-mirror/lsof

**Pull request merged**